static/.htaccess

```diff
@@ -8,6 +8,10 @@ Options -Indexes -MultiViews
 
 # Redirect pages - https://httpd.apache.org/docs/current/mod/mod_rewrite.html#rewriterule
 
+# Modeler start instance and deploy page renaming and merging
+RewriteRule ^docs/components/modeler/web-modeler/start-instance/(.*)$ /docs/components/modeler/web-modeler/run-or-publish-your-process/$1 [R=301,L]
+RewriteRule ^docs/components/modeler/web-modeler/save-and-deploy/(.*)$ /docs/components/modeler/web-modeler/run-or-publish-your-process#deploy-a-process/$1 [R=301,L]
```

Comment:
@pepopowitz @christinaausley can we redirect to page section headers? I remember this wasn't working previously.

Reply:
I'm not sure, I've never tried it! I will try locally and see what happens.

Reply:
Ok, a few things about these rules that I learned!

1. I'm guessing the reason we haven't been able to redirect to page section headers is that the server encodes the `#` character as `%23`. I noticed this when running the redirects locally. 
2. But I learned that [we can prevent this by adding the `NE` flag to the rule](https://serverfault.com/a/987814). (`NE` stands for "no escape" which is very relatable 😅 ) 
3. These redirects are both incorrect for the current PR, as the changes are only made in the `next` version. So they should both be matching and targeting the `next` URLs, instead of the current version (which does not specify a version in the path). 
4. There are also issues with the wildcard matching. For the first rule, the rule works, though it could be improved to eliminate one redirect. The rule matches `start-instance/(.*)`, which means the start-instance page, _and any of its children_. This page doesn't have any children, so that's a minor point, it will just never affect anything but the main page. But there's no need to capture this group, or preserve it.

Reply:
See my novel at https://github.com/camunda/camunda-platform-docs/pull/2058#discussion_r1187957017